README.md

```diff
@@ -62,3 +62,7 @@ qemu-system-x86_64 -bios "${COREBOOT_FILE}" -nographic -serial mon:stdio -m 2G -
   -fw_cfg opt/de.cyberus-technology/cmdline_addr,file=./cmdline_addr \
   -fw_cfg opt/de.cyberus-technology/initrd_size_addr,file=./initrd_size_addr
 ```
+
+By default, QEMU generates ACPI tables and coreboot doesn't overwrite them but only extend them. To debug ACPI-related 
```

Comment:
I think coreboot still needs to create FADT and maybe MADT because they rely on runtime information. Especially the FADT depends on how coreboot configures the chipset. They could just hardcode the 'usual' values, but that would be wrong...

Reply:
I'm not sure what's the right thing here to write. However, it is important that for parity between QEMU and our %FOO% VMM, coreboot needs to set up all of the ACPI tables.